tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "jsx": "react",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "outDir": "./tmp",
```

Comment:
I would have remove the outDir and set `"noEmit": true` do avoid generating files and ignoring everything.

Not an issue, you can resolve